keywords.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/local/bin/python3
+# tested on debian 8.5, you need python-pip to install the following modules:
+# pyyaml, bs4, slackclient, translate
+
+from random import randint
+import random
+from slack_commands import __send_response, __reaction
+
+
+def keywords(sc, evt, yamldata, count):
+    if "channel" in evt and evt["type"] == "message":
+        message = evt["text"].strip()
+        if "bot_id" not in evt:
+            for k in yamldata["keywords"]:
+                if k in message.lower():
+                    # Intent here is not to trigger every time.
+                    # Increment on times seen, trigger a random recount vault
+                    # when we deploy a message
+                    if count["current"] > count["random"]:
+                        # Does our entry only apply to a specific channel?
+                        if yamldata["keywords"][k]["watch"] != "all":
```

Comment:
Maybe
```
if yamldata["keywords"][k]["watch"] != "all" and yamldata["keywords"][k]["watch"] == evt["channel"]
```
?

Or, is there def a need to do this `all` check, vs just
```
if yamldata["keywords"][k]["watch"] == evt["channel"]:
```
?

Reply:
It looks like the content of `if/else` blocks is duplicated. Changing the check probably can help reduce it to one `if` block.